composer.json

```
@@ -18,6 +18,24 @@
         "php-coveralls/php-coveralls": "dev-master"
     },
     "autoload": {
-        "files": ["google/appengine/runtime/autoloader.php"]
+        "psr-4": {
+            "google\\": "src"
```

One more thought! If we are removing the `datastore/` directory ( which I believe we are? ) we should also remove the `appengine` directory, move everything up into `src/` that is beneath it, and change this to:

```suggestion
            "google\\appengine\\": "src"
```

---

Ok I will update this, thanks!